lib/rollbar/capistrano3.rb

```diff
@@ -10,21 +10,21 @@
   desc 'Send deployment started notification to Rollbar.'
   task :deploy_started do
     on primary fetch(:rollbar_role) do
-      ::Rollbar::CapistranoTasks.deploy_started(self, self, dry_run?)
+      ::Rollbar::CapistranoTasks.deploy_started(self, self, false)
```

Comment:
I didn't see/couldn't find a valid `dry_run` option (though I'm sure one exists), so have set to `false` for now.

Reply:
What's the reason behind changing this? `dry_run?` is a method from the Capistrano3 object which should be available in the context of self here. It's set in the command line when running capistrano `--dry-run`.

Reply:
Oh, I see: https://github.com/rollbar/rollbar-gem/issues/832#issuecomment-467522316

That's interesting. I'm investigating this.